### PR TITLE
Add changes for sample 8

### PR DESCRIPTION
**Pull request by matiasroldan6**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

File: samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java [hunk 1]
```diff
@@ -30,14 +34,16 @@
...
+        "This bot will introduce you to suggestedActions. Please answer the question:";
 
     @Override
     protected CompletableFuture<Void> onMembersAdded(
         List<ChannelAccount> membersAdded,
         TurnContext turnContext
     ) {
+        // Send a welcome message to the user and tell them what actions 
+        // they may perform to use this bot
         return sendWelcomeMessage(turnContext);
```
Comment (Batta32) [inline samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java]: Just to confirm, this is correctly accessing to the `static` method? I mean, this method is not `static`, should it be `SuggestedActionsBot.sendWelcomeMessage(turnContext);`?

File: samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java [hunk 2]
```diff
@@ -118,7 +132,7 @@ private Activity createSuggestedActions() {
         SuggestedActions actions = new SuggestedActions();
         actions.setActions(Arrays.asList(redAction, yellowAction, blueAction));
         reply.setSuggestedActions(actions);
-
-        return reply;
+
```
Comment (Batta32) [inline samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java]:
```suggestion

```

File: samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java [hunk 3]
```diff
@@ -51,11 +57,12 @@
 
         // Respond to the user.
         return turnContext
-            .sendActivities(MessageFactory.text(responseText), createSuggestedActions())
-            .thenApply(responses -> null);
+            .sendActivity(MessageFactory.text(responseText))
+            .thenCompose(responses -> sendSuggestedActions(turnContext))
```
Comment (Batta32) [inline samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java]:
```suggestion
            .thenCompose(response -> sendSuggestedActions(turnContext))
```

As it's `sendActivity`

File: samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java [hunk 4]
```diff
@@ -64,11 +71,12 @@
...
             .map(
-                channel -> turnContext.sendActivities(
+                channel -> turnContext.sendActivity(
                     MessageFactory.text(
-                        "Welcome to SuggestedActionsBot " + channel.getName() + ". " + WELCOMETEXT
-                    ), createSuggestedActions()
-                )
+                        "Welcome to SuggestedActionsBot " + channel.getName() + ". " + WELCOME_TEXT
+                    )
+                ).thenCompose(responses -> sendSuggestedActions(turnContext))
```
Comment (Batta32) [inline samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java]:
```suggestion
                ).thenCompose(response -> sendSuggestedActions(turnContext))
```

File: samples/java_springboot/08.suggested-actions/README.md [hunk 5]
```diff
@@ -75,11 +87,14 @@ After the bot is deployed, you only need to execute #6 if you make changes to th
...
+- [Maven Plugin for Azure App Service](https://github.com/microsoft/azure-maven-plugins/tree/develop/azure-webapp-maven-plugin)
 - [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
 - [Activity processing](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-concept-activity-processing?view=azure-bot-service-4.0)
 - [Azure Bot Service Introduction](https://docs.microsoft.com/azure/bot-service/bot-service-overview-introduction?view=azure-bot-service-4.0)
 - [Azure Bot Service Documentation](https://docs.microsoft.com/azure/bot-service/?view=azure-bot-service-4.0)
+- [Azure for Java cloud developers](https://docs.microsoft.com/en-us/azure/java/?view=azure-java-stable)
+- [Bot Framework Documentation](https://docs.botframework.com)
+
+
+
```
Comment (Batta32) [inline samples/java_springboot/08.suggested-actions/README.md]: Remove empty lines

**Review by Batta32 — CHANGES_REQUESTED** [review]
Also, for the README:
1. Check `Test the bot using Bot Framework Emulator` section (see [example](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/java_springboot/05.multi-turn-prompt#testing-the-bot-using-bot-framework-emulator))
2. Check the Further Reading, you are missing some [links](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/08.suggested-actions#further-reading) and also be careful on the order

File: samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java (hunk 1 above)
Comment (matiasroldan6) [reply samples/java_springboot/08.suggested-actions/src/main/java/com/microsoft/bot/sample/suggestedactions/SuggestedActionsBot.java]: I'll follow the [multilingual sample](https://github.com/southworks/BotBuilder-Samples/blob/Java-R13/samples/java_springboot/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java#L75) and call it through the class

**Pull request closed without merging**